src/App.js

```diff
@@ -0,0 +1,85 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { FiTrash, FiPlus } from "react-icons/fi";
+
+function App() {
+
+  const [todos, setTodos] = useState(JSON.parse(localStorage.getItem("todos")) || []);
+  const [newTask, setNewTask] = useState("");
+
+  function addTodos(task) {
+    const todo = { id: nanoid(), task, completed: false };
+    setTodos((prevTodos) => {
+      const newTodo = [todo, ...prevTodos];
+      localStorage.setItem("todos", JSON.stringify(newTodo));
+      return newTodo;
+    });
+  }
+
+  function complete(todoId) {
+    const updatedTask = todos.map((x) =>
+      x.id === todoId ? { ...x, completed: !x.completed } : x
+    );
+    localStorage.setItem("todos", JSON.stringify(updatedTask));
+    setTodos(updatedTask);
+  }
+
+  function deleteTodos(todoId) {
+    const deleteTask = todos.filter((x) => x.id !== todoId);
+    localStorage.setItem("todos", JSON.stringify(deleteTask));
+    setTodos(deleteTask);
+  }
+
+  return (
+    <div>
+      <div className="flex flex-col justify-center items-center h-screen bg-pink-300 overflow-y-scroll">
+        <h1 className="text-green-900 opacity-30 text-9xl content-center">
+          todos
+        </h1>
+        <div className="flex items-center border border-gray-200 rounded-lg h-12 w-1/3 mt-10 bg-slate-100 text-black">
+          <input
+            onChange={(e) => setNewTask(e.target.value)}
+            type="search"
+            className="w-full px-5 bg-transparent outline-none"
+            placeholder="Add todo..."
+          />
```

Comment:
This input is uncontrolled, you should bind its value to the newTask state and I would highly recommend setting newTask to empty string after creating a task and appending it to the todos array.
```suggestion
          <input
            onChange={(e) => setNewTask(e.target.value)}
            type="search"
            className="w-full px-5 bg-transparent outline-none"
            placeholder="Add todo..."
            value={newTask}
          />
```